### DISCUSSION TITLE
Availability of new SDK for Android

### DISCUSSION THREAD

**Discussion by rajagopalannairr** [Azure/azure-mobile-apps#706] [Q&A] [answered]
I just want to check the availability of new SDK for native Android. I am planning to upgrade the old Azure Mobile App to new one.

**Comment by adrianhall** [comment, marked as answer]
I have no intention of writing native clients for this service.

**Comment by rajagopalannairr** [comment]
Does that mean Microsoft is going to support only MAUI in future and those who want to use the services they need to right the custom logic in the native apps?
